app/tasks/build/dev.mjs

```diff
@@ -1,17 +1,15 @@
-import { paths } from 'govuk-frontend-config'
 import { npm } from 'govuk-frontend-tasks'
 import gulp from 'gulp'
 
 import { watch } from '../index.mjs'
 
-import dist from './dist.mjs'
-
 /**
  * Dev task
  * Runs a sequence of tasks on start
+ *
+ * @type {import('govuk-frontend-tasks').TaskFunction}
  */
-export default gulp.series(
-  dist,
-  watch,
-  npm.script('serve', paths.app)
+export default (options) => gulp.parallel(
```

Comment:
Notice that tasks now have an options param?

This is preparation for multiple build targets sharing the same tasks in future

For example in package/tasks we might run the "scripts" task three times:

1. Rollup ES modules build (without polyfills)
2. Rollup ES modules build with polyfills bundled
3. Rollup ES modules build with polyfills as dependencies

By moving build paths into an options parameter we no longer need to hard code them